data/102/2814/en_US.json

```diff
@@ -1,13 +1,13 @@
-{
-	"102": {
-		"2814": {
-			"name": "BANNED_DEVICE_IN_NEX_SERVICE",
-			"message": "",
-			"short_description": "Your console is permanently banned from this game",
-			"long_description": "Unknown cause.",
-			"short_solution": "Don't do whatever you did to get banned",
-			"long_solution": "Unknown solution.",
-			"support_link": "https://preten.do/102-2814"
-		}
-	}
+{
+	"102": {
+		"2814": {
+			"name": "BANNED_DEVICE_IN_NEX_SERVICE",
+			"message": "Online services cannot be used.\n\nPlease make note of the error code\nand visit support.nintendo.com.",
+			"short_description": "Your console is permanently banned from this game",
+			"long_description": "Unknown cause.",
+			"short_solution": "Don't do whatever you did to get banned",
+			"long_solution": "Unknown solution.",

```

Comment:
This is my fault not yours, but since you're changing these files anyway would you mind updating the `long` fields in cases like this? There's multiple of them